packages/go_router/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Update README
```

Comment:
Can you also update the change log?

Reply:
But I already made this:

`- Hiding the internal implementation from external API.`

Reply:
Oh, i got this.
I should add new version to changelog.

Reply:
Move whole section of `NEXT RELEASE` to `5.1.2`
```Markdown
## 5.1.2

- Hiding the internal implementation from external API.
- Update README
- Removes dynamic calls in examples.
```